### PR TITLE
Add KMEM DAX to glossary

### PR DESCRIPTION
**Pull request by michalbiesek**
Preview: https://michalbiesek.github.io/glossary/#kmem-dax

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)

### CODE REVIEW

**Review by lukaszstolarczuk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @michalbiesek)

___
*[glossary/index.md, line 713 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MagsWid-InjsliBpMRE:-MagsWid-InjsliBpMRF:b8br07q) ([raw file](https://github.com/pmem/pmem.github.io/blob/f2bd941b6ff36cd0cba30cd9cdcf00cab87e4697/glossary/index.md#L713)):*
> ```Markdown
> For a volatile use of persistent memory, there is an alternative for [Memory Mode](#memory-mode).
> [Device DAX](#device-dax) can be configured in [system-ram mode](https://pmem.io/ndctl/daxctl-reconfigure-device.html).
> This mode allows to exposed PMem as a hotplugged memory region - **KMEM DAX**.
> ```

`allows to exposed` -> `allows exposing` or `allows us to expose`
___
*[glossary/index.md, line 714 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MagsikZ7L_ltLKWqzd2:-MagsikZ7L_ltLKWqzd3:bnqqjie) ([raw file](https://github.com/pmem/pmem.github.io/blob/f2bd941b6ff36cd0cba30cd9cdcf00cab87e4697/glossary/index.md#L714)):*
> ```Markdown
> [Device DAX](#device-dax) can be configured in [system-ram mode](https://pmem.io/ndctl/daxctl-reconfigure-device.html).
> This mode allows to exposed PMem as a hotplugged memory region - **KMEM DAX**.
> Persistent memory takes a form of separate memory-only NUMA Node(s). In contrast to
> ```

of **a** separate

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @lukaszstolarczuk)

___
*[glossary/index.md, line 713 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MagsWid-InjsliBpMRE:-MahNAUS857MYZ5HxkRt:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/f2bd941b6ff36cd0cba30cd9cdcf00cab87e4697/glossary/index.md#L713)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

`allows to exposed` -> `allows exposing` or `allows us to expose`
</blockquote></details>

Done.
___
*[glossary/index.md, line 714 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MagsikZ7L_ltLKWqzd2:-MahNAlPBq2sibBNwoc1:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/f2bd941b6ff36cd0cba30cd9cdcf00cab87e4697/glossary/index.md#L714)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

of **a** separate
</blockquote></details>

Done.

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)* status: 1 of 3 files reviewed, 1 unresolved discussion (waiting on @lukaszstolarczuk)

___
*[glossary/kmem-dax/_definition, line 7 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/155#-Mahkmf1ByiZxNfNyjqq:-Mahoi2F1XQeHkM3xW6m:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/209a93ec6d9b7786792da83e4b6653602881554d/glossary/kmem-dax/_definition#L7)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

I think you're missing the EOF line (here and below) - other files have that
</blockquote></details>

Done.

**Review by lukaszstolarczuk — COMMENTED** [review]
:lgtm:

> Reviewed 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155#-:-Mahq6TD7_JhExOLIhuS:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @michalbiesek)

___
*[glossary/kmem-dax/_definition, line 7 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/155#-Mahkmf1ByiZxNfNyjqq:-Mahq2Fu-IXjyCpGAcQs:b2nid0f) ([raw file](https://github.com/pmem/pmem.github.io/blob/209a93ec6d9b7786792da83e4b6653602881554d/glossary/kmem-dax/_definition#L7)):*
<details><summary><i>Previously, michalbiesek wrote…</i></summary><blockquote>

Done.
</blockquote></details>

thx

File: glossary/kmem-dax/_definition [hunk 1]
```diff
@@ -0,0 +1,7 @@
+For a volatile use of persistent memory, there is an alternative for [Memory Mode](#memory-mode).
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:1]: KMEM DAX is a semi-transparent alternative to Memory Mode for volatile use of PMem.

File: glossary/kmem-dax/_definition [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+[Device DAX](#device-dax) can be configured in [system-ram mode](https://pmem.io/ndctl/daxctl-reconfigure-device.html).
+This mode allows exposing PMem as a hotplugged memory region - **KMEM DAX**.
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:3]: This mode exposes

File: glossary/kmem-dax/_definition [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+Persistent memory takes a form of a separate memory-only NUMA Node(s). In contrast to
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:4]: Persistent Memory takes the form of

File: glossary/kmem-dax/_definition [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+**Memory Mode** - PMem is represented as independent memory resources
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:5]: comma (`,`) instead of `-`

is exposed as an

File: glossary/kmem-dax/_definition [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+explicitly managed by the Operating System, not implicitly by the memory controller.
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:6]: drop `not implicitly by the memory controller`

File: glossary/kmem-dax/_definition [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+More information can be found on the [KMEM DAX blog post](https://pmem.io/2020/01/20/memkind-dax-kmem.html).
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:7]: For more information about KMEM DAX, see ...

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)* status: 1 of 3 files reviewed, 6 unresolved discussions (waiting on @lukaszstolarczuk, @PatKamin, and @pbalcer)

___
*[glossary/kmem-dax/_definition, line 1 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-1:-Mb6B4HY6Oog3gIES1U2:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L1)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

KMEM DAX is a semi-transparent alternative to Memory Mode for volatile use of PMem.
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 3 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-3:-Mb6Ao517ayRvZ9ceeEH:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L3)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

This mode exposes
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 4 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-4:-Mb6AQUK0OqYuibszjgP:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L4)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

Persistent Memory takes the form of
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 5 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-5:-Mb6Aebo5O78ONovy4uV:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L5)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

comma (`,`) instead of `-`

is exposed as an
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 6 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-6:-Mb6AFHc08t-D4OAwjIr:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L6)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

drop `not implicitly by the memory controller`
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 7 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r3-7:-Mb69vE6D6fuvVaBplen:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/a53d63f5408fb0c6b3220e018c87453d5d5d57e1/glossary/kmem-dax/_definition#L7)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

For more information about KMEM DAX, see ...
</blockquote></details>

Done.

File: glossary/kmem-dax/_definition [hunk 7]
```diff
@@ -0,0 +1,6 @@
+**KMEM DAX** is a semi-transparent alternative to [Memory Mode](#memory-mode) for volatile use of PMem.
+[Device DAX](#device-dax) can be configured in [system-ram mode](https://pmem.io/ndctl/daxctl-reconfigure-device.html).
+This mode exposes PMem as a hotplugged memory region - KMEM DAX.
+Persistent Memory takes the form of a separate memory-only NUMA Node(s). In contrast to **Memory Mode**,
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:4]: Reading this now - it doesn't flow well.
`This mode exposes PMem as a hotplugged memory region. Configured this way, Persistent Memory takes the form of a separate memory-only NUMA Node(s).`

File: glossary/kmem-dax/_definition [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+PMem is represented as independent memory resources explicitly managed by the Operating System.
```
Comment (pbalcer) [inline glossary/kmem-dax/_definition:5]: "as an independent"

**Review by michalbiesek — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/155)* status: 1 of 3 files reviewed, 8 unresolved discussions (waiting on @lukaszstolarczuk, @PatKamin, and @pbalcer)

___
*[glossary/kmem-dax/_definition, line 4 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r4-4:-Mb6OEBU6IAFCLjv8KeV:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/740285e68ae3aa5af0eca4b57c735941bc6c1d6e/glossary/kmem-dax/_definition#L4)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

Reading this now - it doesn't flow well.
`This mode exposes PMem as a hotplugged memory region. Configured this way, Persistent Memory takes the form of a separate memory-only NUMA Node(s).`
</blockquote></details>

Done.
___
*[glossary/kmem-dax/_definition, line 5 at r4](https://reviewable.io/reviews/pmem/pmem.github.io/155#-MadowWvqsvV_d2dv6be-r4-5:-Mb6OF9d5s_8_XcYKhJP:b-896fix) ([raw file](https://github.com/pmem/pmem.github.io/blob/740285e68ae3aa5af0eca4b57c735941bc6c1d6e/glossary/kmem-dax/_definition#L5)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

"as an independent"
</blockquote></details>

Done.

**Pull request merged**